### PR TITLE
Update gir files

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: pango/src/auto/font_map.rs [hunk 1]
```diff
@@ -19,6 +19,26 @@ impl FontMap {
 }
 
 pub trait FontMapExt: IsA<FontMap> + 'static {
+    #[cfg(feature = "v1_56")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v1_56")))]
+    #[doc(alias = "pango_font_map_add_font_file")]
+    fn add_font_file(&self, filename: &str) -> Result<(), glib::Error> {
```
Comment (bilelmoussaoui) [inline pango/src/auto/font_map.rs:25]: should this be path?

Comment (sdroege) [reply pango/src/auto/font_map.rs:25]: Yes, will send an MR. Thanks!

Comment (sdroege) [reply pango/src/auto/font_map.rs:25]: Seems like gitlab is broken currently

Comment (sdroege) [reply pango/src/auto/font_map.rs:25]: https://gitlab.gnome.org/GNOME/pango/-/merge_requests/775

Comment (sdroege) [reply pango/src/auto/font_map.rs]: Done

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**